libs/sr/formula/src/data/char/sheets/RuanMei.ts

```diff
@@ -51,6 +53,7 @@ const sheet = register(
     { ...baseTag, damageType1: 'break' },
     subscript(char.talent, dm.talent.breakDmg)
   ),
+  // TODO: shouldn't this be incorporated to the talentBreakDmg above?
```

Comment:
why?

Reply:
is e6 a separate instance of dmg from the talent hit? i read it as an increase in multiplier (same instance of dmg)

Reply:
True it should be added